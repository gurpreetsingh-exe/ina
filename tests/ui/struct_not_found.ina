// COMPILE_FAIL

fn main() {
    let a = Foo { a: 32 };
}

