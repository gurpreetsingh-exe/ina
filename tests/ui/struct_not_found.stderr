error: struct `Foo` is not found in this scope
   --> struct_not_found.ray:4:13
    |
 LL |     let a = Foo { a: 32 };
    |             ^^^^^^^^^^^^^ not found in this scope

